superset/dashboards/api.py

```diff
@@ -1225,3 +1228,68 @@ def delete_embedded(self, dashboard: Dashboard) -> Response:
         for embedded in dashboard.embedded:
             DashboardDAO.delete(embedded)
         return self.response(200, message="OK")
+
+    @expose("/<id_or_slug>/copy/", methods=["POST"])
+    @protect()
```

Comment:
let's make this endpoint permission be `can write on Dashboard` add copy_dash to https://github.com/apache/superset/blob/master/superset/constants.py#L151